examples/fedprox_example/config.yaml

```diff
@@ -16,7 +16,7 @@ local_epochs: 1 # The number of epochs to complete for client
 batch_size: 128 # The batch size for client training
 
 reporting_config:
-  enabled: True
+  enabled: False
```

Comment:
I assume you're setting this to false to make testing a bit less annoying? I'm good with that, just wanted to make sure it was intentional.

Reply:
Yeah, just because by default we dont have a valid entity name. I just set it to False so it does not run.